snapshots/output/basic-project/lib/more.dart

```diff
@@ -77,6 +76,13 @@
       }
     }
   
+    factory Animal.cat() => Animal('Timmy', type: AnimalType.cat);
```

Comment:
Very frustratingly, in named constructors, `Animal.cat()` there is only a single identifier

![Screenshot 2024-12-19 at 10 57 04 AM](https://github.com/user-attachments/assets/3d92d2a5-5eea-4e9f-a46b-ac989f418387)

This isn't the case for SimpleIdentifiers for the call sight, but only the constructor declarations

The result of this, is we can only index a single element here, and I've decided to index, the arguably more important, definition for `cat()`

```dart
factory Animal.cat() => ...
//      ^^^^^^ NOT INDEXED
//             ^^^ definition ... Animal#cat().
```

Solving this is a problem for another day...

Reply:
I created a ticket to track this here: https://github.com/Workiva/scip-dart/issues/166